app/emails/email.py

```diff
@@ -13,4 +13,11 @@ def send_password_reset_email(user):
     send_email('Stock Trader: Reset Your Password',
                sender='jerry.aragon@student.csulb.edu', recipients=[user.email],
                text_body=render_template('email/reset_password.txt', user=user, token=token),
-               html_body=render_template('email/reset_password.html',user=user, token=token))
\ No newline at end of file
+               html_body=render_template('email/reset_password.html',user=user, token=token))
+
+def send_notification_email(changes_made, user):
```

Comment:
Why is changes_made been passed as the argument?

Reply:
changes_made tracks what attributes where changed in the settings page, it is passed into the method when changes were made in the settings page.